internal/controller/apiextensions/composite/composition_functions_test.go

```diff
@@ -515,7 +561,23 @@ func TestFunctionCompose(t *testing.T) {
 			reason: "We should return a valid CompositionResult when a 'pure Function' (i.e. patch-and-transform-less) reconcile succeeds",
 			params: params{
 				kube: &test.MockClient{
-					MockGet:         test.NewMockGetFn(kerrors.NewNotFound(schema.GroupResource{Resource: "UncoolComposed"}, "")), // all names are available
+					// MockGet:
+					// test.NewMockGetFn(kerrors.NewNotFound(schema.GroupResource{Resource:
+					// "UncoolComposed"}, "")), // all names are available
```

Comment:
are these commented out lines still needed?

Reply:
Nope, they were left in by accident. Thanks!